R/data-importer-configuration.R

```diff
@@ -220,9 +220,9 @@ DataImporterConfiguration <- R6::R6Class(
       rClr::clrCall(private$.dataImporterTask, "SetAllLoadedSheet", self$ref, value)
     },
     #' @field namingPattern Regular expression used for naming of loaded data sets.
-    #' Words between curly brackets (e.g. \code{{Group Id}}) will be replaced by the value
-    #' in the corresponding column. Further keywords are \code{{Source}} for the file name
-    #' and \code{{Sheet}} for sheet name.
+    #' Words between curly brackets (e.g. `Group Id` will be replaced by the value
```

Comment:
Can you please add the curly brackets back?  E.g. `{Group Id}`

Reply:
![image](https://user-images.githubusercontent.com/11330453/156329869-c8a24e64-9b51-4ac2-8330-ffe70f6b1ae6.png)